optimum_benchmark/backends/base.py

```diff
@@ -119,7 +134,9 @@ def check_initial_isolation(self) -> None:
             cuda_devices = os.environ.get("CUDA_VISIBLE_DEVICES")
             if cuda_devices is None:
                 LOGGER.warning(
-                    "Asked to check the initial device isolation, but the variable CUDA_VISIBLE_DEVICES was not set. Defaulting to checking on the first device."
+                    "Asked to check the initial device isolation, "
+                    "but the variable CUDA_VISIBLE_DEVICES was not set. "
+                    "Defaulting to checking on the first device."
```

Comment:
Are you using ruff or anything? I think it can make sense to add a workflow to check the style (like in optimum & transformers), and a Makefile to make style.